packages/cli/templates/react/igr-ts/projects/_base/files/index.html

```diff
@@ -8,7 +8,7 @@
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
     <meta name="theme-color" content="#000000" />
-    <title>IgniteUI for React</title>
+    <title><%=ApplicationTitle%></title>
```

Comment:
this causes the placeholder to appear in other templates (empty and top-nav) because they extend base with home that doesn't have its own index.html which means that it will reuse the one from _base containing the placeholder
![image](https://github.com/IgniteUI/igniteui-cli/assets/19392175/57f1a9da-cee7-4225-90f8-1cba46a6a385)

Reply:
A new index.html with 'IgniteUI for React' title is added in packages\cli\templates\react\igr-ts\projects\_base_with_home\files. 
This way it overrides the one from base and won't leak the <%=ApplicationTitle%> placeholder to other templates anymore.